proto/tendermint/p2p/pex.proto

```diff
@@ -3,18 +3,23 @@ package tendermint.p2p;
 
 option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
 
-import "tendermint/p2p/types.proto";
 import "gogoproto/gogo.proto";
 
+message PexAddress {
+  string id   = 1 [(gogoproto.customname) = "ID"];
+  string ip   = 2 [(gogoproto.customname) = "IP"];
+  uint32 port = 3;
```

Comment:
uint16?

Reply:
There is no uint16 in Protobuf, only 32 and 64.